aesthetic_ants/penguin/main.py

```diff
@@ -1,9 +1,22 @@
 import pyglet
 
+from .player import Player
+from .utils import keys
+
 window = pyglet.window.Window(caption='Penguin Snowball')
 
+player = Player(window.width / 2, window.height / 2)
+
+
+@window.event
+def on_draw():
+    window.clear()
+    player.draw()
+
 
 def main():
+    window.event(player.on_mouse_motion)  # Registers the player-mouse event handler
```

Comment:
```suggestion
    window.push_handlers(player)  # Registers the player event handlers
```

Up to you whether or not to implement this but it looks more consistent with the `window.push_handlers(keys)`

Reply:
Integrated 👍